### PR TITLE
Fixed an issue where screen readers will wrongly read files and folders as Files.Filesystem.ListedItem

### PR DESCRIPTION
**Pull request by jaigak**
Note: This doesn't work in DataGrid. I will open another pull request for data grid list view layout mode.

### CODE REVIEW

File: Files/Strings/en-US/Resources.resw [hunk 1]
```diff
@@ -1152,4 +1152,16 @@
...
+  </data>
+  <data name="FolderItemAutomation" xml:space="preserve">
+    <value>folder</value>
+  </data>
+  <data name="RecycleBinItemAutomation" xml:space="preserve">
+    <value>recycle bin item</value>
+  </data>
+  <data name="ShortcutItemAutomation" xml:space="preserve">
+    <value>shortcut item</value>
+  </data>
```
Comment (yaichenbaum) [inline Files/Strings/en-US/Resources.resw:1166]:
```suggestion
  <data name="FileItemAutomation" xml:space="preserve">
    <value>File</value>
  </data>
  <data name="FolderItemAutomation" xml:space="preserve">
    <value>Folder</value>
  </data>
  <data name="RecycleBinItemAutomation" xml:space="preserve">
    <value>Recycle bin item</value>
  </data>
  <data name="ShortcutItemAutomation" xml:space="preserve">
    <value>Shortcut item</value>
  </data>
```

**Comment by yaichenbaum** [comment]
Can you please update the genereated resource files?

File: Files/MultilingualResources/Files.ja-JP.xlf [hunk 2]
```diff
@@ -1424,7 +1440,7 @@
         </trans-unit>
         <trans-unit id="SettingsWidgetsShowRecentFiles.Header" translate="yes" xml:space="preserve">
           <source>Show recent files on the home page</source>
-          <target state="new">Show recent files on the home page</target>
+          <target state="new">Show recent files on the home page</targe
```
Comment (yaichenbaum) [inline Files/MultilingualResources/Files.ja-JP.xlf:1443]:
```suggestion
          <target state="new">Show recent files on the home page</target>
```

File: Files/MultilingualResources/Files.nl-NL.xlf [hunk 3]
```diff
@@ -1400,6 +1400,22 @@
...
+          <source>folder</source>
+          <target state="new">folder</target>
+        </trans-unit>
+        <trans-unit id="RecycleBinItemAutomation" translate="yes" xml:space="preserve">
+          <source>recycle bin item</source>
+          <target state="new">recycle bin item</target>
+        </trans-unit>
+        <trans-unit id="ShortcutItemAutomation" translate="yes" xml:space="preserve">
+          <source>shortcut item</source>
+          <target state="new">shortcut item</target
```
Comment (yaichenbaum) [inline Files/MultilingualResources/Files.nl-NL.xlf:1417]:
```suggestion
          <target state="new">shortcut item</target>
```

File: Files/MultilingualResources/Files.or-IN.xlf [hunk 4]
```diff
@@ -1436,7 +1452,7 @@
         </trans-unit>
         <trans-unit id="SettingsWidgetsShowRecentFiles.Header" translate="yes" xml:space="preserve">
           <source>Show recent files on the home page</source>
-          <target state="new">Show recent files on the home page</target>
+          <target state="new">Show recent files on the home page</targe
```
Comment (yaichenbaum) [inline Files/MultilingualResources/Files.or-IN.xlf:1455]:
```suggestion
          <target state="new">Show recent files on the home page</target>
```

File: Files/MultilingualResources/Files.pt-BR.xlf [hunk 5]
```diff
@@ -1394,7 +1410,7 @@
           <source>Drives</source>
           <target state="new">Drives</target>
         </trans-unit>
-        <trans-unit id="SettingsWidgetsShowLibraryCards.AutomationProperties.Name" translate="yes" xml:space="preserve">
+        <trans-unit id="SettingsWidgetsShowLibraryCards.AutomationProperties.Name" translate="yes" xml:space="preserve">11
```
Comment (yaichenbaum) [inline Files/MultilingualResources/Files.pt-BR.xlf:1413]:
```suggestion
        <trans-unit id="SettingsWidgetsShowLibraryCards.AutomationProperties.Name" translate="yes" xml:space="preserve">
```

**Review by yair100 — CHANGES_REQUESTED** [review]

File: Files/MultilingualResources/Files.or-IN.xlf [hunk 6]
```diff
@@ -1397,7 +1397,23 @@
         <trans-unit id="SettingsOpenItemsWithOneclick.Header" translate="yes" xml:space="preserve">
           <source>Open items with a single click</source>
           <target state="new">Open items with a single click</target>
-        </trans-unit>
```
Comment (yaichenbaum) [inline Files/MultilingualResources/Files.or-IN.xlf:1400]: It looks like this was removed by mistake in several files.

**Comment by yaichenbaum** [comment]
@Jaiganeshkumaran Can you fix the generated files so that they are up to date?

**Review by yair100 — DISMISSED** [review]

**Comment by yaichenbaum** [comment]
@Jaiganeshkumaran Can you solve the merge conflicts?

**Comment by jaigak** [comment]
@yair100 Done

**Review by yair100 — APPROVED** [review]

**Pull request merged**